### ISSUE TITLE
PDFUA-179: Figure with space as Alt text | FAIL

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-7#179]
Jira issue originally created by user Peter Spraul <p.spraul@axes4.com>:

This example demonstrates the failure condition that applies for an image that conveys information, but the relevant information is not conveyed by the [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text). In this case there is only a space set as [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text).
 
Please note: A simple machine check will not return an error message related to missing [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) here.

**Comment by zakkinsey** [comment]
Peter Spraul `<p.spraul@axes4.com>` assigned issue to self

**Comment by zakkinsey** [comment]
@MarkusErle changed assignee from Peter Spraul `<p.spraul@axes4.com>` to self

**Comment by zakkinsey** [comment]
Comment created by @MarkusErle:

Cleaned up with Roman's tool

**Comment by zakkinsey** [comment]
@MarkusErle added a link to #178: This issue relates to #178

**Comment by zakkinsey** [comment]
@zakkinsey unassigned issue from @MarkusErle as part of jira-&gt;github migration
